.rhcicd/pr_check_fp.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -exv
+
+# Clowder config
+export APP_NAME="policies"
+export COMPONENT_NAME="policies-ui-backend"
+export IMAGE="quay.io/cloudservices/iqe-tests:policies"
+export DEPLOY_TIMEOUT="600"
+
+# IQE plugin config
+export IQE_PLUGINS="floorist"
+export IQE_MARKER_EXPRESSION="floorist_smoke"
+export IQE_FILTER_EXPRESSION=""
+export IQE_CJI_TIMEOUT="30m"
+
+# Bonfire init
+CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
+curl -s $CICD_URL/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_bootstrap.sh
+
+# Build the image and push to Quay
+export DOCKERFILE=src/main/docker/Dockerfile.jvm
+source $CICD_ROOT/build.sh
+
+# Deploy on ephemeral
+export COMPONENTS_W_RESOURCES="policies-engine"
+source $CICD_ROOT/deploy_ephemeral_env.sh
```

Comment:
Running this standalone would require another environment to be spawn, which increases resource utilization.